sgkit/io/vcf/vcf_reader.py

```diff
@@ -341,6 +344,11 @@ def vcf_to_zarr(
         have a ``.csi`` index file.
     output
         Zarr store or path to directory in file system.
+    target_part_size
```

Comment:
Should this have a non-None default value?  It reminds me a bit of the per-worker memory limit used in rechunker that I didn't make into a required parameter.  Anyways, I think we could be opinionated on it if you have a feeling for a what a good setting is and since other target chunk settings have defaults.  Is there any other reason a user might prefer synchronous processing?

Reply:
Parallel processing by default would probably be a good idea. But I don't have a great feeling for what a good setting is yet. I've used 1GB for processing the Ag1000G data and each part took around 5 minutes to process, which seems OK for larger (~100GB) VCF files. OTOH for more modest-sized files it would still be nice to have some parallelism, so 100MB might be a better default.

Reply:
I've changed the default to 100MB, and updated the documentation to reflect the change.

Reply:
I'd argue it's better to not encode the default in the signature though - we can easily imagine a future where we try to find a good value, or the best value depends on the inputs. Using a local sentinel via ``DEFAULT_TARGET_PART_SIZE=object()`` would allow us to detect when the user doesn't specify a value as well as supporting the None semantics here.

Reply:
+1 Perhaps we could have a string value like `"auto"` - then we could also allow human-readable byte sizes like `"1GB"` etc.

Reply:
Latest commit supports human-readable strings and `"auto"` is the default, see https://github.com/pystatgen/sgkit/pull/412/commits/502511bef08fed037a7a8c681e54ad5a9cec8024